spec/factories/course_groups.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :course_group do
+    sequence(:name) { |n| "Course Group #{n}" }
+
+    initialize_with do
```

Comment:
Not a fan of this as it's making `FactoryBot.create` a bit misleading. I'd rather see some `#find_by` in the specs really - no harm in relying on seeds in specs.

Reply:
Removed `initialize_with` and added `find_by` 👍